platform/darwin/scripts/generate-style-code.js

```diff
@@ -519,13 +522,29 @@ global.setSourceLayer = function() {
     return `_layer->setSourceLayer(sourceLayer.UTF8String);`
 };
 
+const lightProperties = Object.keys(spec[`light`]).reduce((memo, name) => {
```

Comment:
Nit: a backtick-delimited [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) is only necessary for inserting `${variables}`. Otherwise, a simple `'string'` will do.